pyke/tests/test_lightcurve.py

```diff
@@ -67,3 +69,31 @@ def test_lightcurve_plot():
     lcf = KeplerLightCurveFile(TABBY_Q8)
     lcf.plot()
     lcf.SAP_FLUX.plot()
+
+
+def test_sff_detrender():
+    fn = get_pkg_data_filename('./data/ep60021426alldiagnostics.csv')
```

Comment:
Let's make sure we add a comment to explain where this csv file comes from and what it contains.